src/content.js

```diff
@@ -1,192 +0,0 @@
-/**
- * All of the items in the NugetFunctionality namespace indicate that the functions come from the Nuget Gallery JavaScript.
```

Comment:
I just changed jQuery magic back to template strings because it would be probably easier for you to change them if needed and I might not be around to tweak jQuery wizardry... :wink: